src/snowflake/connector/secret_detector.py

```diff
@@ -70,7 +71,7 @@ def mask_private_key_data(text):
         return SecretDetector.PRIVATE_KEY_DATA_PATTERN.sub('"privateKeyData": "XXXX"', text)
 
     @staticmethod
-    def mask_secrets(text: str) -> str:
+    def mask_secrets(text: str) -> Tuple[bool, str, str]:
```

Comment:
`telemetry.py` looks like expect this returns a str.

Reply:
+1

Reply:
Thats a bug then. The secret detector has always returned a tuple. I'll fix that in telemetry.py and look for other misuses

Reply:
That might be a wrong usage of telemetry.py. Could you please help to fix that in telemetry.py? @sfc-gh-kwagner

Reply:
@sfc-gh-stakeda @sfc-gh-wshangguan Fixed